scripts/minikube.sh

```diff
@@ -99,6 +99,18 @@ ssh)
     echo "connecting to minikube"
     minikube ssh
     ;;
+deploy-rook)
+    echo "deploy rook"
+    ./scripts/rook.sh deploy
+    ;;
+clean-rook)
+    echo "clean rook"
+    ./scripts/rook.sh clean
+
+    # delete rook data for minikube
+    minikube ssh "sudo rm -rf /mnt/${DISK}/var/lib/rook"
```

Comment:
can you please check is this directory correct?

Reply:
in the minkube.sh, we have following code to create a link so the default dataDirHostPath will work.
```
minikube ssh "sudo mkdir -p /mnt/${DISK}/var/lib/rook; sudo ln -s /mnt/${DISK}/var/lib/rook /var/lib/rook"
```

I want to delete both link(ln -s) and real directory (mkdir -p), and recreate it.